modules/ztp-configuring-the-hub-cluster-for-backup-and-restore.adoc

```diff
@@ -0,0 +1,177 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp_far_edge/ztp-preparing-the-hub-cluster.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ztp-configuring-the-hub-cluster-for-backup-and-restore_{context}"]
+= Configuring the hub cluster for backup and restore
+
+You can use {ztp} to configure a set of policies to backup `BareMetalHost` resources. 
```

Comment:
```suggestion
You can use {ztp} to configure a set of policies to backup `BareMetalHost` resources. 
```
I know this already went through peer review and this is a cherry pick, but when used as a verb, "back up" [should be two words](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#b).

Reply:
Thanks for reviewing @kcarmichael08